randovania/games/samus_returns/pickup_database/pickup-database.json

```diff
@@ -336,7 +346,7 @@
             "default_shuffled_count": 1,
             "default_starting_count": 0,
             "preferred_location_category": "major",
-            "original_location": 144
+            "original_location": 12
```

Comment:
AFAIK original locations dont work on progressives.

Reply:
This is Morph Ball

Reply:
I was tricked by morph ball not immediately showing in the diff, disregard this.